### ISSUE TITLE
Calculator in launcher - using "." instead of "," gives no result for regions using decimal comma

### ISSUE THREAD

**Issue by MyNameIsArko** [pop-os/shell#1450]
**(1) Issue/Bug Description:**

When using "." as separator instead of ",", calculator will show empty block
![Zrzut ekranu z 2022-05-26 15-37-55](https://user-images.githubusercontent.com/47687092/170499545-eaaf5a34-07cc-4a84-8dd5-30173b68bfbb.png)

**(2) Steps to reproduce (if you know):**

Hit super and enter 0.5 . Calculator will show empty block

**(3) Expected behavior:**

Calculator should display 1/2

**(4) Distribution (run `cat /etc/os-release`):**
 ```
NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os
```

**(5) Gnome Shell version:**

41.2

**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**

```
pop-shell:
  Installed: 1.1.0~1652994129~22.04~0358b15
  Candidate: 1.1.0~1652994129~22.04~0358b15
  Version table:
 *** 1.1.0~1652994129~22.04~0358b15 1001
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages
       1001 http://apt.pop-os.org/release jammy/main i386 Packages
        100 /var/lib/dpkg/status
```

**(7) Where was Pop Shell installed from:**

With pop os installation.

**(8) Monitor Setup (2 x 1080p, 4K, Primary(Horizontal), Secondary(Vertical), etc):**

Laptop screen only.

**(9) Other Installed/Enabled Extensions:**

None

**(10) Other Notes:**

**Comment by mmstick** [comment]
Would you happen to be in a region where `,` is interpreted as `.`?

**Comment by MyNameIsArko** [comment]
Yes, kinda. In Poland we use "," and "." interchangeably. Every other thing will give in calculator some info displayed, but "," will give blank panel.

**Comment by mmstick** [comment]
The mode for qalculate changes based on that region format, so `,` is required.

**Comment by MyNameIsArko** [comment]
I see qalculate has option for localization and also it's reading LC_LOCALE file to determine separator. Maybe it could be possible to use that for selecting "." as separator in appropriate regions.
Another, maybe simpler option would be to automatically change "." to "," in input before giving it to qalculate.
Is "." used for something else in launcher calculator when typing equation with "="?

Edit: gnome-shell allows calculation with "." and "," in activities, so if "." is not used for anything else it would be great if it was also interpreted for calculator.

**Comment by MyNameIsArko** [comment]
Also using "." can be default choice for people using matlab daily.
